README.md

```diff
@@ -1,47 +1,56 @@
 # Data Submission Service Supplier Frontend
 
+This project can be run locally in two ways - with Docker or without. For speed, we recommend running _without_ Docker.
+
+## Running with Docker
+
+See [Running with Docker](docs/running-with-docker.md)
+
+## Without Docker
+
 ### Prerequisites
- - [Docker](https://docs.docker.com/docker-for-mac) greater than or equal to `18.03.1-ce-mac64 (24245)`
+
+#### DataSubmissionServiceAPI
+
+The [DataSubmissionServiceAPI](https://github.com/dxw/DataSubmissionServiceAPI/pull/245) project _must_ be set up and running before attempting to set up this project.
```

Comment:
Only one question: is there a reason to link to the PR rather than the homepage of the project?

Reply:
No that was a mistake!! 🤦‍♂️